### DISCUSSION TITLE
How would I document something like a container server?

### DISCUSSION THREAD

**Discussion by Riprock** [netbox-community/netbox#16979] [Q&A]
So I have a container server with multiple services and basically I have multiple DNS names that go to this one IP. I know I can put a DNS name for an IP so would I just use the hostname dns name and not any of the service names?

**Comment by a084ed22** [comment]
We have a similar situation with vhosts on a webserver. We opted for filling in the name of the machine, while the various FQDNs that also point to it - typically as CNAME's - are in the notes.
